mungegithub/mungers/inactive-review-handler_test.go

```diff
@@ -0,0 +1,282 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	//"fmt"
+	githubapi "github.com/google/go-github/github"
+	"k8s.io/test-infra/mungegithub/features"
+	"k8s.io/test-infra/mungegithub/github"
+	github_test "k8s.io/test-infra/mungegithub/github/testing"
+	//"net/http"
+	//"k8s.io/kubernetes/pkg/util/sets"
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+const (
+	John            = "John" //author
+	Ken             = "Ken"  //assignee, potential owner
+	Lisa            = "Lisa" //assignee, potential owner
+	Max             = "Max"  //potential owner, new reviewer
+	positiveComment = "The changes look great!"
+	negativeComment = "The changes break things!"
+	firstFile       = "first-file.go"
+	firstFileTest   = "first-file_test.go"
+	baseDir         = ""
+	leafDir         = "a/b/c"
+)
+
+func TestInactiveReviewHandler(t *testing.T) {
+	runtime.GOMAXPROCS(runtime.NumCPU())
+
+	//testRepo := getTestRepo()
+	//testAliases := getTestAliases()
+	tests := []struct {
+		name        string
+		issue       *githubapi.Issue
+		comments    []*string
+		commitFiles []*githubapi.CommitFile
+		feats       *features.Features
+		expected    string
+	}{
+		{
+			name:        "obj.issue = nil",
+			issue:       nil,
+			comments:    []*string{},
+			commitFiles: []*githubapi.CommitFile{},
+			expected:    "",
+		},
+		{
+			name: "is an issue",
```

Comment:
That got me confused somehow. Maybe should be "Is not a PR" (because PRs are also issues somehow ...)